src/openloco/objects/tree_object.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include "../types.hpp"
+
+namespace openloco
+{
+#pragma pack(push, 1)
+    struct tree_object
```

Comment:
You can use https://github.com/jawsper/locotool/blob/master/libloco/objects.py#L697 to name some unknown fields. There's also https://www.tt-wiki.net/wiki/Variable:List#Class_18_-_Trees, but not very useful in this case

Reply:
Thanks, I had been using the tt-wiki but like you said there wasn't much information there for trees.